src/Helpers/Forms.php

```diff
@@ -0,0 +1,87 @@
+<?php
+
+/**
+ * Helpers for forms
+ *
+ * @package EightshiftForms\Helpers
+ */
+
+declare(strict_types=1);
+
+namespace EightshiftForms\Helpers;
+
+/**
+ * Helpers for forms
+ */
+class Forms
+{
+
+	/**
+	 * Overrides the value from $_GET if it's set.
+	 *
+	 * @param  string $value Input field's value.
+	 * @param  string $name  Field's key / name.
+	 * @return string
+	 */
+	public static function maybeOverrideValueFromPost(string $value, string $name)
+	{
+
+		/**
+		 * Ignoring nonce verification missing warning because of the dynamic nature of this feature
+		 * ( i.e. you can set any form to post to another form and prefill any field )
+		 * not sure how nonce could be implemented here.
+		 */
+    // phpcs:disable WordPress.Security.NonceVerification.Missing
```

Comment:
Did you try to make nonce dependent on the `$name` parameter?

That way you'd have something like: `wp_create_nonce("form-$name-nonce");`? The name parameter will be passed to it, the same way you are providing it here. Just an idea 🤷🏼‍♂️

Reply:
Good idea, in order to get this working simple with regular POST forms it was done like this, tho the form could pass the nonce as value. Originally this was envisioned to be used in a way a client could share the URL with pre-filled data (but that was scrapped because of GDPR issues and now only works with POST).

WIll look into it but probably not as part of this PR.